src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        int result;
+        String lineSplitter = ",";
+        String supplyString = "supply";
```

Comment:
these 2 variables are local variables, not constants
constants are declared at class level